pkg/reconciler/ingress/resources/destinationrule.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	istiov1alpha3 "istio.io/api/networking/v1alpha3"
+	"istio.io/client-go/pkg/apis/networking/v1alpha3"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/networking/pkg/apis/networking/v1alpha1"
+	"knative.dev/pkg/kmeta"
+)
+
+const (
+	// has to match config/700-istio-secret.yaml
+	knativeServingCertsSecret = "knative-serving-certs"
+
+	// has to match https://github.com/knative-sandbox/control-protocol/blob/main/pkg/certificates/constants.go#L21
```

Comment:
I think David is making this a public constant shortly.

Reply:
Will use that as soon as available (in this PR or a followup depending on merge order).

Reply:
We don't have a dependency on `control-protocol` here and introducing it just for the constant is a bit much. I heard that we maybe are  going to refactor the certificate part out of `control-protocol` into `networking`. If that is so, we can keep it and use it from `networking` when it is done?

Reply:
I think it is better to use the `control-protocol` even adding the deps. Otherwise we have to keep in mind of this place.